miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditBoxController.java

```diff
@@ -176,4 +196,37 @@ public String processSubmit(@ModelAttribute("box") Box box, ModelMap model, Sess
     model.clear();
     return "redirect:/miso/box/" + box.getId();
   }
+
+  private final class BulkCreateBoxBackend extends BulkCreateTableBackend<BoxDto> {
+    public BulkCreateBoxBackend(Integer quantity) {
+      super("box", BoxDto.class, "Boxes", new BoxDto(), quantity);
+    }
+
+    @Override
+    protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) throws IOException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)